js/pages/myToDoList.js

```diff
@@ -862,5 +850,12 @@ const setModuleAttributes = (data, modules, collections) => {
         modules['PROMIS'].completed = true;
     }
 
+    if (data[fieldMapping.verifiedDate] && +new Date(data[fieldMapping.verifiedDate]) > +new Date('2024-03-01T00:00:00.000Z') && +new Date(data[fieldMapping.verifiedDate]) < +new Date('2024-07-31T23:59:59.999Z')) {
```

Comment:
If existed, verifiedDate is in ISO string format. So comparing strings should be more straightforward.

Reply:
I'm going to stick with this for now because I'm not sure if Brian had some particular reason for this method of comparison.